docs/source/getting-started/docker-machine-drivers.adoc

```diff
@@ -101,13 +101,33 @@ $ sudo chown root:wheel $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/doc
 $ sudo chmod u+s $(brew --prefix)/opt/docker-machine-driver-xhyve/bin/docker-machine-driver-xhyve
 ----
 
-For more information, see the GitHub documentation of the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver].
+For more information, see the GitHub documentation for the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver].
 
 [[manual-install]]
 === Manual install
 
 To manually install the xhyve driver, run the following commands:
 
+. Download the `docker-machine-driver-xhyve` binary using:
```

Comment:
Nowhere we mention to where we should download the driver. There are really two options for that. Either you download it to the location of the other docker-machine drivers which is for OS X (which we are targeting here) _/usr/local/bin/docker-machine-driver-xhyve_. Alternatively one needs to make sure that docker-machine-driver-xhyve is on the PATH. Either by downloading it to another directory which is on the PATH or by explicitly adding the directory one wants to keep  docker-machine-driver-xhyve to the PATH.

Either way, the key is that the driver needs to be on the PATH, otherwise it is not going to work.

If you compile from source _make install_ is probably taking care of this.